app/views/citizens/additional_accounts/index.html.erb

```diff
@@ -0,0 +1,43 @@
+<% content_for :navigation do %>
+  <%#= link_to 'Back', path-to-be-determined, class: 'govuk-back-link', id: 'back' %>
+<% end %>
+
+  <%= form_tag(citizens_additional_accounts_path) do %>
```

Comment:
**[not a blocker]** One of the things we discussed offline as a team was that there was quite a lot of repetition across the multiple form pages we have in the app.

💭 It would be 👍 to slowly try to reduce that repetition, things like:

```html
    <div class="govuk-form-group">
      <fieldset class="govuk-fieldset">
        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
          <h1 class="govuk-fieldset__heading">
            Do you have accounts with other banks?
          </h1>
        </legend>
        <br />
       ...
```

are the same across all form pages and only the heading copy changes (which could even be set in the locales).

Also, this file and https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/64/files#diff-e7165391fd39abd65998b4a03e647975 seem very similar which only re-enforces this point. DRY.

Reply:
OK